Dockerfile

```diff
@@ -0,0 +1,30 @@
+# ===============================
+# [ Builder image ]
+# ===============================
+FROM openjdk:11-jdk-slim AS BUILDER
+
+ARG WORK_DIR=/meeteor13
+
+COPY . $WORK_DIR
+WORKDIR $WORK_DIR
+
+RUN ./mvnw -Dmaven.test.skip=true install
+
+# ===============================
+# [ Production image ]
+# ===============================
+FROM openjdk:11-jre-slim
+
+LABEL name="meeteor13-core" \
+      maintainer="aliaksandr.babai@gmail.com"
+
+ARG WORK_DIR=/meeteor13
+
+ENV SERVER_PORT=80 \
+    JAVA_OPTS=""
+
+WORKDIR $WORK_DIR
+COPY --from=BUILDER $WORK_DIR/target/*.jar .
+
+EXPOSE $SERVER_PORT
+ENTRYPOINT ["/bin/sh", "-c", "java $JAVA_OPTS -Djava.security.egd=file:/dev/urandom -jar *.jar"]
```

Comment:
why you add all this shit instead just
`ENTRYPOINT java -jar *.jar`

Reply:
stupid grebec
please read matchast'

Reply:
it will not work.
this shit `*.jar` is processed by shell